### PR TITLE
use getTickCount calculate timeout

### PR DESCRIPTION
**Pull request by tomerf-sndbox**

### CODE REVIEW

File: analyzer/windows/modules/auxiliary/human.py [hunk 1]
```diff
@@ -139,13 +139,15 @@ class Human(threading.Thread, Auxiliary):
     def __init__(self, options={}, analyzer=None):
         threading.Thread.__init__(self)
         Auxiliary.__init__(self, options, analyzer)
+        self.office_close_timeout = round(self.options.get("timeout") * 1000 * 0.666)
```
Comment (1tayH) [inline analyzer/windows/modules/auxiliary/human.py]: what is that 0.666 magic number?

**Comment by tomerf-sndbox** [comment]
2/3 of analysis time

**Comment by 1tayH** [comment]
document the use case and why it's done

File: analyzer/windows/modules/auxiliary/human.py [hunk 2]
```diff
@@ -168,8 +171,9 @@ def run(self):
             self.do_click_buttons = int(self.options["human.click_buttons"])
 
         while self.do_run:
-            if seconds and not seconds % 60:
+            if not is_office_close and KERNEL32.GetTickCount() > end:
                 USER32.EnumWindows(EnumWindowsProc(get_office_window), 0)
+                is_office_close = True
```
Comment (1tayH) [inline analyzer/windows/modules/auxiliary/human.py]: I think `break` would be a better fit

**Comment by tomerf-sndbox** [comment]
The human loop can still run. For example if malware is monitoring the
mouse behavior.

**Pull request merged**